fog/view/server/src/sharding_strategy.rs

```diff
@@ -140,4 +167,80 @@ mod epoch_sharding_strategy_tests {
 
         assert!(!should_process_block)
     }
+
+    #[test]
+    fn is_ready_to_serve_tx_outs_first_epoch_processed_block_count_is_zero_less_than_50_percent_processed_returns_true(
+    ) {
+        // The first epoch has a start block == 0.
+        const START_BLOCK: BlockIndex = 0;
+        const END_BLOCK_NON_INCLUSIVE: BlockIndex = 100;
```

Comment:
Looks like #2350 changed to `END_BLOCK_EXCLUSIVE` might consider it here as well since they show up in the test cases above

Reply:
Done.